src/aat/resources/features/F-063/F-063.feature

```diff
@@ -0,0 +1,45 @@
+@F-063
+Feature: F-063: Retrieve search input details for dynamic display
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-215
+  Scenario: should retrieve search inputs
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains a valid case type]
+    And it is submitted to call the [Retrieve search input details for dynamic display] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [contains the correct search inputs for the given case type, along with an HTTP 200 OK]
```

Comment:
[contains the correct search **results** for the given case type, along with an HTTP 200 OK]

Reply:
it isn't the search results that are returned, it is a list of search inputs that are allowed for that case type... in other words, the different aspects of a case type that can be searched for (eg. defendant name, solicitor name, etc.)

Reply:
Makes sense.